src/components/common/SummaryDefineCompany.vue

```diff
@@ -20,6 +24,32 @@
         </span>
       </div>
 
+      <template v-if="isAmalgamationFiling">
+        <article class="section-container pb-0">
+          <v-row no-gutters>
+            <v-col
+              cols="12"
+              sm="3"
+              class="pr-4"
+            >
+              <label id="amalgamation-type-label">Amalgamation Type</label>
+            </v-col>
+            <v-col
+              cols="12"
+              sm="9"
+              class="pt-4 pt-sm-0"
+            >
+              <div id="amalgamation-type">
+                <span v-if="isAmalgamationFilingRegular">Regular</span>
+                <span v-else-if="isAmalgamationFilingHorizontal">Horizontal</span>
+                <span v-else-if="isAmalgamationFilingVertical">Vertical</span>
+                <span v-else>[Unknown]</span>
```

Comment:
Anyone want to argue that this should be a computed value? Or shall I leave it?

Reply:
I think it's probably fine as is